### DISCUSSION TITLE
안녕하세요. 티스토리 hELLO 사용하다가 갑자기 무한로딩이 발생해서 여쭤봅니다.

### DISCUSSION THREAD

**Discussion by JungH200000** [pronist/hELLO#263] [Q&A] [answered]
1달정도 hELLO를 잘 쓰고 있었는데 오늘 갑자기 티스토리 블로그 무한로딩이 발생해서 다른 스킨으로 변경해 보았더니 정상적으로 작동이 되더라고요. 혹시 이때 어떻게 해야하는지 알 수 있을까요?

**Comment by pronist** [comment, marked as answer]
@JungH200000 해당 버그 처리하여 4.10.2 버전 배포하였습니다. 스킨 내부의 코드에서 발생한 문제이므로 다시 다운받으셔서 적용하시면 됩니다.

**Reply by JungH200000** [reply]
감사합니다!!!! 항상 잘 사용하고 있습니다!
